### DISCUSSION TITLE
How to configure the analytics for mkdocs-material

### DISCUSSION THREAD

**Discussion by cooper-lzy** [squidfunk/mkdocs-material#3949] [Q&A]
My configuration is as follows, but the web does not seem to receive the configuration.
![image](https://user-images.githubusercontent.com/78672629/170190220-be6c32e8-dc5f-4916-b98a-3b0cdb50689f.png)
![image](https://user-images.githubusercontent.com/78672629/170190286-c22647ff-8f22-4a05-83a1-879dc32b905e.png)
![image](https://user-images.githubusercontent.com/78672629/170190327-26f7d5e8-d737-4713-9aef-241ebe7eb5be.png)

My web page will generate analytics.js, but there is no data.
![image](https://user-images.githubusercontent.com/78672629/170190504-40b203d8-fd39-45d8-8a2c-bcf6012346a1.png)

How to modify the above configuration for analytics to take effect? HELP!

**Comment by squidfunk** [comment]
The key must be named [`extra.analytics`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#google-analytics), not `extra.google_analytics`.
